Vagrantfile

```diff
@@ -10,38 +10,62 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb|
-    vb.memory = 12288
-    vb.cpus = 4
-  end
-
-  # https://app.vagrantup.com/folio/boxes/snapshot-backend-core
-  config.vm.define "snapshot-backend-core", autostart: false do |snapshot_backend_core|
-    snapshot_backend_core.vm.box = "folio/snapshot-backend-core"
-  end
-
-  # https://app.vagrantup.com/folio/boxes/snapshot-core
-  config.vm.define "snapshot-core", autostart: false do |snapshot_core|
-    snapshot_core.vm.box = "folio/snapshot-core"
+    vb.memory = 36864
+    vb.cpus = 8
   end
 
   # https://app.vagrantup.com/folio/boxes/snapshot
   config.vm.define "snapshot", autostart: false do |snapshot|
     snapshot.vm.box = "folio/snapshot"
-  end
 
-  # https://app.vagrantup.com/folio/boxes/testing-backend
-  config.vm.define "testing-backend", autostart: false do |testing_backend|
-    testing_backend.vm.box = "folio/testing-backend"
-  end
+    snapshot.vm.synced_folder "C:/Users/William/Development/work/FOLIO/fw-cli", "/home/vagrant/fw-cli", owner: "vagrant", group: "vagrant", mount_options: ["uid=1000", "gid=1000"]
```

Comment:
please replace this path `C:/Users/William/Development/work/FOLIO/fw-cli` with generic/relative path